db/migrate/20170501201633_add_canceled_to_order_detail.rb

```diff
@@ -4,10 +4,11 @@ def up
     add_column :order_details, :canceled_by, :integer
     add_column :order_details, :canceled_reason, :string
 
-    Reservation.where.not(order_detail_id: nil).find_each do |reservation|
-      reservation.order_detail.update_attributes(canceled_at: reservation.canceled_at,
+    Reservation.where.not(order_detail_id: nil, canceled_at: nil).find_each do |reservation|
+      reservation.order_detail.assign_attributes(canceled_at: reservation.canceled_at,
```

Comment:
I'm inclined to agree with Meara in this case. I would consider this change to be an integral part of the migration, and needs to happen on every row across every environment. If you're updating more specific rows or specific environments, then a rake task is better.